integration_test.go

```diff
@@ -2634,3 +2634,118 @@ func TestNewTableFromSnapshot(t *testing.T) {
 		t.Fatalf("expected no cells after RestoreSnapshot in table %s key %s", table, key)
 	}
 }
+
+// TestScannerTimeout tests whether there is a lease timeout between Next calls
+// if the wait between them is too long. This is fixed by the renewal process
```

Comment:
nit: I think the purpose of this test is to make sure when the Renew() flag is off, we are indeed getting the UnknownScannerException error. So maybe change the comment to something like 
```
// TestScannerTimeout makes sure that without the Renew flag on, we get a lease timeout between Next calls
// if the wait between them is too long. 
```

Reply:
Changed the comment, thanks!